nevergrad/benchmark/experiments.py

```diff
@@ -1139,6 +1140,21 @@ def images(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
                         yield xp
 
 
+@registry.register
+def imagesUsingGan(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
```

Comment:
Following pep8, this should be:
```suggestion
def images_using_gan(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
```